src/app/components/dashboard/dashboardView.html

```diff
@@ -1,3 +1,289 @@
+                    <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
+                  </button>
+                  <button class="btn btn-primary">
+                    <i class="fa fa-eye" aria-hidden="true"></i> View
+                  </button>
+                  <button class="btn btn-danger">
+                    <i class="fa fa-close" aria-hidden="true"></i> Leave
+                  </button>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td>Face Painting</td>
+              <td>SGA</td>
+              <td>10/20/2017</td>
+              <td>University of Central Florida</td>
+              <td>34.1 MI</td>
+              <td>
+                <div class="btn-group-vertical">
+                  <button class="btn btn-success">
+                    <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
+                  </button>
+                  <button class="btn btn-primary">
+                    <i class="fa fa-eye" aria-hidden="true"></i> View
+                  </button>
+                  <button class="btn btn-danger">
+                    <i class="fa fa-close" aria-hidden="true"></i> Leave
+                  </button>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td>Face Painting</td>
+              <td>SGA</td>
+              <td>10/20/2017</td>
+              <td>University of Central Florida</td>
+              <td>34.1 MI</td>
+              <td>
+                <div class="btn-group-vertical">
+                  <button class="btn btn-success">
+                    <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
+                  </button>
+                  <button class="btn btn-primary">
+                    <i class="fa fa-eye" aria-hidden="true"></i> View
+                  </button>
+                  <button class="btn btn-danger">
+                    <i class="fa fa-close" aria-hidden="true"></i> Leave
+                  </button>
+                </div>
+              </td>
+            </tr>
+            </tbody>
+          </table>
+        </div>
+        <div class="panel-footer">
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-5 col-md-offset-1">
+      <div class="row">
+        <div class="col-md-12">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="text-center">Make RSO:</h3>
+            </div>
+            <div class="panel-body">
+              <div class="col-md-6">
+                <div class="form-group">
+                  <label for="rso-name">RSO Name:</label>
+                  <input type="text" class="form-control" id="rso-name">
+                </div>
+              </div>
+              <div class="col-md-6">
+                <div class="form-group">
+                  <label for="rso-lead-email">RSO Leader Email:</label>
```

Comment:
Rereading the docs, i think you need 5 people PLUS you. Ill double check though.